packages/shared/src/graphql/actions.ts

```diff
@@ -12,6 +12,7 @@ export enum ActionType {
   BrowserExtension = 'browser_extension',
   EnableNotification = 'enable_notification',
   WritePost = 'write_post',
+  HideBlockPanel = 'hide_block_panel',
```

Comment:
We need to add this to `UserActionType` on API side to keep enums in sync.

Reply:
Currently unused in the backend. Not sure if we have to. @rebelchris WDYT?